server/db/models/review.js

```diff
@@ -0,0 +1,23 @@
+const Sequelize = require('sequelize')
+const {INTEGER, TEXT} = Sequelize
+const db = require('../db')
+
+const Review = db.define('review', {
+  content: {
+    type: TEXT,
+    allowNull: false,
+    len: {
+      args: [20] //this should be min characters
+    }
+  },
+  rating: {
+    type: INTEGER,
+    validate: {
```

Comment:
maybe allowNull and just have min: 1?
and default null?
nbd

Reply:
Changed min to 1 and added allowNull to true which should make default null.  In the latest merge.